rllib/evaluation/sampler.py

```diff
@@ -587,7 +587,7 @@ def _process_policy_eval_results(to_eval, eval_results, active_episodes,
             env_id = eval_data[i].env_id
             agent_id = eval_data[i].agent_id
             if clip_actions:
```

Comment:
It's better to use bare functions instead of a class if possible (principle of least power)

Reply:
Ok, it is stateless. We should move it to utils, though.

Reply:
Moved back.